### PR TITLE
data: mrdanishsaleem

### PR DESCRIPTION
**Pull request by mrdanishsaleem**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5248"></a>

### CODE REVIEW

File: data/mrdanishsaleem.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Danish Saleem",
+  "type": "personal",
+  "bio": "Software Engineer 💻",
+  "avatar": "https://github.com/mrdanishsaleem.png"
```
Comment (kumarsonsoff3) [inline data/mrdanishsaleem.json:5]: `avatar` key is no longer needed. So please remove this from your json. 

```suggestion
```

File: data/mrdanishsaleem.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/SaraJaoude",
```
Comment (kumarsonsoff3) [inline data/mrdanishsaleem.json:10]: Update it with your Twitter link. 🛠

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by mrdanishsaleem — COMMENTED** [review]
🐛: The `avatar` key is removed
🔗: Twitter link updated

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Review by amandamartin-dev — APPROVED** [review]

**Pull request merged**